framework/include/ble_manager/ble_common.h

```diff
@@ -69,11 +69,17 @@ typedef struct {
 	ble_addr_type_e type;
 } ble_addr;
 
+typedef enum {
+	SLAVE_CONN_PARAM_UPDATE,
```

Comment:
Are these enums used anywhere ?
If not these can be removed

Reply:
@thapav The enum name is changed to match the naming convention.